### ISSUE TITLE
android.permission.READ_NETWORK_USAGE_HISTORY

### ISSUE THREAD

**Issue by Oisins** [francoiscampbell/Xposed-DataUsage#6]
After launching the app android.systemui crashed. The log shows the following error:

```
SecurityException: NetworkStats: Neither user 10013 nor current process has
android.permission.READ_NETWORK_USAGE_HISTORY 
```

Specs:
Samsung J1 (Rooted) 
android 4.4

**Comment by francoiscampbell** [comment]
Hmm, odd. Seems SystemUI doesn't have that permission on every ROM. Thanks for the report, I'll investigate a solution.

**Comment by francoiscampbell** [comment]
It's fixed in 1.0, try it now. If you don't see it in the Xposed app, you can get it from the releases section.

**Comment by Oisins** [comment]
Can confirm. Great work and very fast!
